src/internal/testing/TestScheduler.ts

```diff
@@ -24,378 +44,508 @@ interface FlushableTest {
   expected?: any[];
 }
 
+export interface TestObservable<T> extends Observable<T> {
+  subscriptions: SubscriptionLog[];
+  messages: TestMessage<T>[];
+}
+
 export type observableToBeFn = (marbles: string, values?: any, errorValue?: any) => void;
 export type subscriptionLogsToBeFn = (marbles: string | string[]) => void;
 
 export class TestScheduler extends VirtualTimeScheduler {
-  public readonly hotObservables: HotObservable<any>[] = [];
-  public readonly coldObservables: ColdObservable<any>[] = [];
-  private flushTests: FlushableTest[] = [];
-  private runMode = false;
+  private _runMode = false;
+  private _flushTests: FlushableTest[] = [];
+  // private _patches = DEFAULT_SCHEDULER_PATCHES;
```

Comment:
If this - and the associated interface - isn't used, can we get rid of it? Personally, I'm not a fan of commented-out code. Either that or mark it with a TODO comment.